gtest sources are copied twice (on Ubuntu)

As we can see in [build 351](https://buildkite.com/fawkesrobotics/fawkes-build/builds/351#58acc994-3bac-431e-b510-3392f3f83ed6), the googletest sources are (sometimes) copied twice into the source tree, which causes some builds to fail:
```
--> Entering sub-directory src
   --> Entering sub-directory src/libs
      --> Entering sub-directory src/libs/googletest
      --> Entering sub-directory src/libs/protobuf_comm
         [CPY] /usr/src/googletest/googletest/src/gtest-internal-inl.h -> src/gtest-internal-inl.h
         [CPY] /usr/src/googletest/googletest/include/gtest -> src/libs/googletest/gtest
         [CPY] /usr/src/googletest/googletest/src/gtest-filepath.cc -> src/gtest-filepath.cpp
         [CPY] /usr/src/googletest/googletest/src/gtest-death-test.cc -> src/gtest-death-test.cpp
         [CPY] /usr/src/googletest/googletest/src/gtest-port.cc -> src/gtest-port.cpp
         [CPY] /usr/src/googletest/googletest/src/gtest-printers.cc -> src/gtest-printers.cpp
         [CPY] /usr/src/googletest/googletest/src/gtest-test-part.cc -> src/gtest-test-part.cpp
         [CPY] /usr/src/googletest/googletest/src/gtest-typed-test.cc -> src/gtest-typed-test.cpp
         [CPY] /usr/src/googletest/googletest/src/gtest.cc -> src/gtest.cpp
         [CPY] /usr/src/googletest/googletest/src/gtest_main.cc -> src/gtest_main.cpp
         [CPY] /usr/src/googletest/googletest/src/gtest-test-part.cc -> src/gtest-test-part.cpp
         [CPY] /usr/src/googletest/googletest/src/gtest-typed-test.cc -> src/gtest-typed-test.cpp
         [CPY] /usr/src/googletest/googletest/src/gtest.cc -> src/gtest.cpp
cp: cannot create regular file '/workdir/src/libs/googletest/src/gtest-typed-test.cpp': File exists
/workdir/src/libs/googletest/Makefile:36: recipe for target '/workdir/src/libs/googletest/src/gtest-typed-test.cpp' failed
make[4]: *** [/workdir/src/libs/googletest/src/gtest-typed-test.cpp] Error 1
```

This does not occur when building with `-j1`.

I had a quick look but couldn't find the issue yet.

--- a/src/libs/googletest/Makefile
+++ b/src/libs/googletest/Makefile
@@ -44,7 +44,8 @@ $(SRCDIR)/src/%.h: /usr/src/googletest/googletest/src/%.h
 	$(SILENT)mkdir -p $(@D)
 	$(SILENT)cp -af $< $@
 
-$(OBJS_all): | $(SRCDIR)/gtest $(addprefix $(SRCDIR)/,$(SRCS_libfawkesgtest) $(SRCS_libfawkesgtest_main))
+$(OBJS_all): | $(SRCDIR)/gtest
+$(OBJS_all): %.o: $(SRCDIR)/%.cpp
 
 $(SRCDIR)/gtest: /usr/src/googletest/googletest/include/gtest
 	$(eval BUILT_PARTS += $@)

